src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.vb

```diff
@@ -1295,19 +1295,24 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Private Function RewriteFloatingToIntegralConversion(node As BoundConversion, typeFrom As TypeSymbol, underlyingTypeTo As TypeSymbol) As BoundExpression
             Debug.Assert(typeFrom.IsFloatingType() AndAlso underlyingTypeTo.IsIntegralType())
             Dim result As BoundExpression = node
+            Dim operand = node.Operand
 
-            Dim mathRound As MethodSymbol
-            ' Call Math.Round method to enforce VB style rounding.
+            ' CInt(Fix(number)) and the like can be simplified to just truncate the number to the integral type
+            If TypeOf operand Is BoundCall Then
```

Comment:
>TypeOf operand Is BoundCall [](start = 15, length = 27)

Consider replacing this with a kind check, otherwise the type check is done twice